Description: Fix PDF compilation
Author: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/1016323
Last-Update: 2022-08-19

Index: gnuplot/src/encoding.c
===================================================================
--- gnuplot.orig/src/encoding.c
+++ gnuplot/src/encoding.c
@@ -380,7 +380,7 @@ latex_input_encoding(enum set_encoding_i
 	break;
     case S_ENC_UTF8:
 	/* utf8x (not utf8) is needed to pick up degree and micro signs */
-	inputenc = "utf8x";
+	inputenc = "utf8";
 	break;
     case S_ENC_INVALID:
 	int_error(NO_CARET, "invalid input encoding used");
Index: gnuplot/docs/gnuplot.doc
===================================================================
--- gnuplot.orig/docs/gnuplot.doc
+++ gnuplot/docs/gnuplot.doc
@@ -5757,7 +5757,6 @@ Ffigure_labels2
  one of them to each point in a plot based on the value in data column 3:
 
    set encoding utf8
-   symbol(z) = "∙□+⊙♠♣♡♢"[int(z):int(z)]
    splot 'file' using 1:2:(symbol($3)) with labels
 
  This example shows use of labels with variable rotation angle in column 4 and
@@ -13305,7 +13304,7 @@ Ffigure_multiple_keys
  generate axis tic labels. The C library routine always use a hyphen character
  (ascii \055) to indicate a negative number, as in -7.  Many people prefer a
  different typographic minus sign character (unicode U+2212) for this purpose,
- as in −7.  The command
+ as in 7.  The command
 
       set minussign
 
